bigtable-hbase-1.x-parent/bigtable-hbase-1.x-mapreduce/pom.xml

```diff
@@ -98,6 +98,11 @@ limitations under the License.
       <artifactId>hbase-server</artifactId>
       <version>${hbase.version}</version>
     </dependency>
+    <dependency>
+      <groupId>com.google.cloud.bigtable</groupId>
+      <artifactId>bigtable-metrics-api</artifactId>
+      <version>${project.version}</version> <!-- {x-version-update:bigtable-client-core-parent:current} -->
+    </dependency>
```

Comment:
please rebase and remove this dep:
mapreduce is meant to be a leaf artifact and shades everything (or everything except hbase & hadoop after $2895). So it doesnt need to declare this dep. Please remove it from the dep plugin below as well